docs/topics/db/models.txt

```diff
@@ -48,7 +48,9 @@ The above ``Person`` model would create a database table like this:
 Some technical notes:
 
 * The name of the table, ``myapp_person``, is automatically derived from
-  some model metadata but can be overridden. See :ref:`table-names` for more
+  some model metadata but can be overridden, normally are automatically
+  generated by combining the name of the app (``myapp``) and the lowercase
+  name of the model (``person``) . See :ref:`table-names` for more
```

Comment:
Suggest:

By default, the name of the table (``myapp_person``) is automatically derived from model metadata, combining the name of the app (``myapp``) and the lowercase name of the model (``person``) .

Reply:
@evildmp addressed in 6a55b66facfc2d421c796c7c1e1609d67c506df0